.buildkite/gem-push.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
Nitpick

```suggestion
#!/bin/base -eu
```

or a comment on why we're not using the standard syntax (we should put on a post somewhere, maybe mobile.blog ? on why we use that)

Reply:
@mokagio I assume you meant `/bin/bash`?

Also for the `e` flag, I understand that it makes sense in most cases, but there are cases where it's OK & necessary to let the script handle the errors and decide whether to continue or not. I am OK with us defaulting to `eu` flags as a team, but if this was a personal project, I'd only use them when necessary and go with the defaults. 🤷

Reply:
> @mokagio I assume you meant /bin/bash?

Yep. Thank you. I updated the snipped.

Regarding the rest of your comment. My understanding is the same. There are some scripts in which we want to handle errors manually, therefore `#!/bin/bash -e` or `set -e` would not be appropriate. The same goes for `-u` for unassigned variables: we might want to prompt the user about them instead of failing.

I think my suggestion is appropriate in the context of this script, but I'll let @AliSoftware weight in.

Also, maybe my representation of what's our desired standard was incorrect. Should we simply go with "use `bash` instead of `sh`?"